shoop/front/middleware.py

```diff
@@ -92,6 +93,9 @@ def _set_timezone(self, request):
             # TODO: Fallback to request.shop.timezone (and add such field)
 
     def _get_maintenance_response(self, request):
+        if resolve(request.path).app_name == "shoop_admin":
```

Comment:
What if resolution fails? Also, I don't like the implication that URL resolution will be done twice per request...